### DISCUSSION TITLE
what is time control and how does it work

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14403] [How to] [answered]
Does anyone have a idea what is time control and how to use it ? i see it in the employee module and projects too..
what does it do ? how can it track users productivity ?

**Comment by diderich** [comment, marked as answer]
It allows to track time spend by employees on specific activities (projects, help desk tickets, ...).

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski its not answered man... question also says how to use it...  do i need to manually add time control  entry for tasks, or can it automatically calculate based on start time... like the moment from which i accept it till marking it as complete..

@diderich really appreciate your time and try to help attitude man...

**Comment by rioshaz** [comment]
@diderich   does it work automatically or do agents or users need to ad did manually how much time they took to cmplete the task ?
and will it show all total at the end in their employee section ?

**Reply by rioshaz** [reply]
pls say
